src/bgw/job.c

```diff
@@ -44,6 +44,8 @@
 #include "bgw/scheduler.h"
 
 #include <cross_module_fn.h>
+#include "jsonb_utils.h"
+#include "unistd.h"
```

Comment:
What do you need this for?
```suggestion
#include <unistd.h>
```

Reply:
I don't need it, I probably added it for debugging and then forgot to remove. Removing it now.